DemoServer/Controllers/Demos/MultiMapIndexes/MultiMapReduceIndex/MultiMapReduceIndexController.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Threading.Tasks;
+using DemoCommon.Models;
+using DemoServer.Utils.Cache;
+using DemoServer.Utils.Database;
+using DemoServer.Utils.UserId;
+using Microsoft.AspNetCore.Mvc;
+#region Usings
+using System.Linq;
+using Raven.Client.Documents.Linq;
+using System.Collections.Generic;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Indexes;
+#endregion
+
+namespace DemoServer.Controllers.Demos.MultiMapIndexes.MultiMapReduceIndex
+{
+    public class MultiMapReduceIndexController : DemoCodeController
+    {
+        public MultiMapReduceIndexController(UserIdContainer userId, UserStoreCache userStoreCache, MediaStoreCache mediaStoreCache,
+            DatabaseSetup databaseSetup) : base(userId, userStoreCache, mediaStoreCache, databaseSetup)
+        {
+        }
+        
+        #region Demo
+        #region Step_1
+        public class CityCommerceDetails : AbstractMultiMapIndexCreationTask<CityCommerceDetails.IndexEntry>
+        #endregion
+        {
+            #region Step_2
+            public class IndexEntry
+            {
+                public string CityName { get; set; }
+                public int NumberOfCompaniesInCity { get; set; } 
+                public int NumberOfSuppliersInCity { get; set; }
+                public int NumberOfOrdersShippedToCity { get; set; }
```

Comment:
What do you think about 

- CityCompaniesCount
- CitySuppliersCount
- CityShippedOrdersCount

as a bit shorter names?

Reply:
TBH - I also usually prefer shorter names - 
but in this case, since it is about explaining to others - and not my own code...  
I think the longer version is better than xxxCount because your mind doesn't need to take that moment and think: 
"count of what ?" 
i.e. CityShippedOrdersCount is not straightforward.
MMR is confusing by itself already.